modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C

```diff
@@ -432,6 +467,9 @@ INSFVRhieChowInterpolator::getVelocity(const Moose::FV::InterpMethod m,
   // Return if Rhie-Chow was not requested
   if (m == Moose::FV::InterpMethod::Average)
     return velocity;
+  mooseAssert((m == _velocity_interp_method) || _a_data_provided,
```

Comment:
```suggestion
  mooseAssert(((m == Moose::FV::InterpMethod::RhieChow) && (_velocity_interp_method == Moose::FV::InterpMethod::RhieChow)) || _a_data_provided,
```

Reply:
ah my bad your assert was more restrictive. I ll add it to Rayleigh PR